app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -1531,4 +1535,12 @@ private void startTheMap() {
         mapView.onStart();
         performMapReadyActions();
     }
+
+    public interface  NearbyParentFragmentListener{
```

Comment:
This interface name can be more definitive ie. NearbyParentFragmentInstanceReadyCallback